docs/docs/dev_docs/tutorials/writing_private_voting_contract.md

```diff
@@ -0,0 +1,251 @@
+import Image from '@theme/IdealImage';
+
+# Writing a private voting smart contract in Aztec.nr
+
+In this tutorial we will go through writing a very simple private voting smart contract in Aztec.nr. You will learn about private functions, public functions, composability between them, and state management.
```

Comment:
```suggestion
In this tutorial we will go through writing a very simple private voting smart contract in Aztec.nr. You will learn about private functions, public functions, composability between them, state management and creatively using nullifiers to prevent people from voting twice!
```